scipy/sparse/linalg/_expm_multiply.py

```diff
@@ -673,6 +673,10 @@ def _expm_multiply_interval(A, B, start=None, stop=None, num=None,
         linspace_kwargs['num'] = num
     if endpoint is not None:
         linspace_kwargs['endpoint'] = endpoint
+    reverse = False
+    if start > stop:
+        reverse = True
+        stop, start = start, stop
```

Comment:
Is this line still correct if `endpoint=False` is specified? In this case, `start` and `stop` are treated differently.